tests/integration/linux_bridge_test.py

```diff
@@ -42,38 +45,56 @@
     priority: 32768
 """
 
-BRIDGE_PORT_ETH1_YAML = """
-port:
-  - name: eth1
-    stp-hairpin-mode: false
-    stp-path-cost: 100
-    stp-priority: 32
+BRIDGE_PORT_YAML = """
+stp-hairpin-mode: false
+stp-path-cost: 100
+stp-priority: 32
 """
 
 
-def test_create_and_remove_linux_bridge_with_one_port(eth1_up):
-    bridge_name = 'linux-br0'
-    bridge_state = yaml.load(BRIDGE_OPTIONS_YAML, Loader=yaml.SafeLoader)
-    port_state = yaml.load(BRIDGE_PORT_ETH1_YAML, Loader=yaml.SafeLoader)
-    bridge_state.update(port_state)
+def test_create_and_remove_linux_bridge_with_min_desired_state():
+    bridge_name = TEST_BRIDGE0
+    with _linux_bridge(bridge_name, bridge_state=None) as desired_state:
+        assertlib.assert_state(desired_state)
 
-    with linux_bridge(bridge_name, bridge_state) as desired_state:
+    assertlib.assert_absent(bridge_name)
+
+
+def test_create_and_remove_linux_bridge_with_one_port(eth1_up):
```

Comment:
Is it really best to use  `eth1_up` as a fixture? Having eth1 down as an initial state seems to make more sense to me actually. In the end it should work for both cases I guess.

Reply:
In general, it depends on where you put the line between the test itself and the setup for it.
The setup you assume and the test you question.

In scenarios where eth1 and eth2 are used as slaves, nmstate implicitly sets them to state `up` as part of the metadata processing. Therefore, I think that for these tests it can work both ways.
It does not work like this for tests which do not implicitly enable the interface.

Therefore, I tend to agree that we can do this here.
How about trying this in a separated PR? This PR has not changed the functionality, only adjusted the creating of the config in order to reuse it later on.

Reply:
Refactor port names eth1 and eth2 for readability and
to sync with global bridge name variable.

Make Port eth1 to be globa var PORT0 and port eth2 to be PORT1.

Reply:
Looks like you already done it in #368 . Please clarify if you want me to change this PR or you will keep yours.